tests/api/test_product.py

```diff
@@ -419,7 +419,7 @@ def test_create_product(
     assert data['product']['description'] == product_description
     assert data['product']['isPublished'] == product_isPublished
     assert data['product']['chargeTaxes'] == product_chargeTaxes
-    assert data['product']['taxRate'] == product_taxRate.lower()
+    assert data['product']['taxRate'] == product_taxRate
```

Comment:
I think we should use `TaxRateType.STANDARD.name` (graphql/core/types/money.py) instead of passing hardcoded string

Reply:
D'oh!!!
I will fix it